src/redux/configureStore.js

```diff
@@ -1,11 +1,15 @@
-import { configureStore } from '@reduxjs/toolkit';
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
 import thunk from 'redux-thunk';
+import missionReducer from './missions/missions';
 import rocketsReducer from './rockets/rocketsSlice';
 
+const rootReducer = combineReducers({
+  rockets: rocketsReducer,
+  missions: missionReducer,
+});
```

Comment:
- [OPTIONAL] Instead of using the combine reducer you can add your reducer directly to reducer section on configureReducer, it will look cleaner.. 👍

`const store = configureStore({
  reducer: {
    rockets: rocketsReducer,
    missions: missionReducer,
  },}`